mod-data-export/src/test/resources/firebird/dataexport/features/export.feature

```diff
@@ -455,11 +455,9 @@ Feature: Tests for uploading "uuids file" and exporting the records
     #should return job execution by id and wait until the job status will be 'COMPLETED'
     Given path 'data-export/job-executions'
     And param query = 'id==' + jobExecutionId
-    And retry until response.jobExecutions[0].status == 'COMPLETED'
+    And retry until response.jobExecutions[0].status == 'COMPLETED' && JSON.stringify(response.jobExecutions[0].progress) == '{"exported":1,"total":1,"failed":0}'
```

Comment:
The status may not be updated when the job with the status 'COMPLETED' did. Need to give a retry attempt until the progress will be updated as well.